packages/pagination/src/add-pagination-body.interceptor.ts

```diff
@@ -0,0 +1,136 @@
+import { CallHandler, ExecutionContext, Injectable, NestInterceptor } from '@nestjs/common';
+// tslint:disable-next-line
+import { map } from 'rxjs/operators';
+// tslint:disable-next-line
+import { Observable } from 'rxjs';
+// tslint:disable-next-line
+import { Request } from 'express';
+
+const FIRST_PAGE: number = 1;
+const DEFAULT_LIMIT: number = 200;
+
+/* tslint:disable no-null-keyword */
+
+/**
+ * Configuration options
+ */
+interface PaginationBodyInterceptorOptions {
+  pageName?: string;
+  perPageName?: string;
+  defaultLimit?: number;
+}
+
+/**
+ * PaginationInterceptor
+ */
+@Injectable()
+export class PaginationBodyInterceptor<T> implements NestInterceptor<DataToPaginate<T>, PaginatedData<T>> {
+  private readonly pageName: string;
+  private readonly perPageName: string;
+  private readonly defaultLimit: number;
+
+  constructor(options: PaginationBodyInterceptorOptions) {
+    const { defaultLimit = DEFAULT_LIMIT, pageName = 'page', perPageName = 'limit' } = options;
```

Comment:
Default values must be the same default as the decorator, so they can work together in their default version
```suggestion
    const { defaultLimit = DEFAULT_LIMIT, pageName = 'page', perPageName = 'per_page' } = options;
```